src/QMCWaveFunctions/RotatedSPOs.cpp

```diff
@@ -626,6 +631,69 @@ void RotatedSPOs::log_antisym_matrix(const ValueMatrix& mat, ValueMatrix& output
     }
 }
 
+void RotatedSPOs::log_antisym_matrix(const Matrix<std::complex<RealType>>& mat, Matrix<std::complex<RealType>>& output)
```

Comment:
Could you reduce duplication?

Reply:
This function was a pain...  in contrast to exponentiate_antisym_matrix and its LAPACK::heev call, this function relies on geev, which has a different argument list if its real or complex.  I could make one function if I could use #ifdef QMC_COMPLEX or if constexpr.